client/src/utils/constants.js

```diff
@@ -0,0 +1,6 @@
+export const TOKEN_PAIRS = ["TRB-DAI"];
+
+export const CONTRACT_ADDRESSES = {
+  "4": ["0x4Aba792D41928D4bC65A4b77d840Fc6418940438"],
+  "1": ["0x4Aba792D41928D4bC65A4b77d840Fc6418940438"],
```

Comment:
Can these be pulled in from a config file some where I can provide a list of contract addresses?